src/js/shield_defs.js

```diff
@@ -4208,6 +4211,45 @@ export function loadShields() {
 
   // Ref-specific cases. Each entry should be documented in CONTRIBUTE.md
 
+  shields["CA:NS:S"].overrideByRef = {
+    BdOLSD: {
+      spriteBlank: "shield_ca_ns_s_bdolsd",
+    },
+    CeT: {
+      spriteBlank: "shield_ca_ns_s_cet",
+    },
+    CT: {
+      spriteBlank: "shield_ca_ns_s_ct",
+    },
+    DNISD: {
+      spriteBlank: "shield_ca_ns_s_dnisd",
+    },
+    ET: {
+      spriteBlank: "shield_ca_ns_s_et",
+    },
+    FdLt: {
+      spriteBlank: "shield_ca_ns_s_fdlt",
+    },
```

Comment:
This seems to have a case difference with https://www.openstreetmap.org/relation/11147926

Reply:
Thanks, fixed.